packages/wonder-blocks-timing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@khanacademy/wonder-blocks-timing",
-  "private": true,
+  "private": false,
   "version": "0.0.3",
   "design": "v1",
   "publishConfig": {
```

Comment:
Could you please change this "access" key to "public"?

Reply:
Oh darn. Thanks! I will.